google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineStopAction.java

```diff
@@ -65,11 +65,14 @@ public void run() {
     GeneralCommandLine commandLine = new GeneralCommandLine(
         appEngineHelper.getGcloudCommandPath().getAbsolutePath());
 
-    commandLine.addParameters("preview", "app", "modules", "stop");
+    commandLine.addParameters("preview", "app", "versions", "stop");
+    commandLine.addParameters(versionToStop);
+
+    commandLine.addParameter("--service");
     for(String module : modulesToStop) {
       commandLine.addParameter(module);
     }
```

Comment:
Yea good catch. It looks like only a single service can be given to the stop command. However, the deploy command looks like its still set up to return multiple modules (services). So I need to determine if this would ever happen, and if so how do we deal with it.